lib/harvesting/models/project_user_assignments.rb

```diff
@@ -0,0 +1,40 @@
+module Harvesting
+  module Models
+    class ProjectUserAssignments < Base
+      include Harvesting::Enumerable
+      extend Forwardable
+
+      attributed :id,
+                 :project,
+                 :user,
+                 :is_active,
+                 :is_project_manager,
+                 :hourly_rate,
+                 :budget,
+                 :created_at,
+                 :updated_at
+
+      def initialize(ref_project, attrs, opts = {})
+        super(attrs.reject {|k,v| k == "user_assignments" }, opts)
+        @api_page = attrs
+        @entries = attrs["user_assignments"].map do |entry|
+          ProjectUserAssignment.new(ref_project, entry, client: opts[:client])
+        end
+      end
+
+      def page
+        @attributes['page']
+      end
+
+      def size
+        total_entries
+      end
```

Comment:
@benphelps Maybe you could refactor this method into `Harvesting::Enumerable`? I see that it is repeated as is `page`

Reply:
@etagwerker We've proposed a design change that would further eliminate the duplication here, as well as the need for so much boilerplate code.

Reply:
@benphelps There is a new section in our README.md that explains a bit about VCR. You should use a test account when developing this gem. This is very important because the tests will delete all your data from the Harvest account that you are using.